tmva/sofie/inc/TMVA/RModel_BasicBinaryOp.hxx

```diff
@@ -11,20 +11,50 @@ namespace TMVA{
 namespace Experimental{
 namespace SOFIE{
 
-template <typename T>
-class ROperator_Add final : public ROperator
-{
+enum EBasicBinaryOperator { Add, Sub, Mul, Div };
 
```

Comment:
You need to make it a template class, add this line:
```
template <typename T, EBasicBinaryOperator Op>
```

Reply:
And the class should not be called `RModel_BasicBinaryOp`, but to be consistent:
`ROperator_BasicBinary`